routes/api/v1.js

```diff
@@ -1079,6 +1079,88 @@ api.declare({
 });
 
 
+/** Report task canceled */
+api.declare({
+  method:     'post',
+  route:      '/task/:taskId/runs/:runId/canceled',
+  name:       'reportCanceled',
+  scopes: [
+    [
+      'queue:resolve-task',
+      'assume:worker-id:<workerGroup>/<workerId>'
+    ]
```

Comment:
I see some copy/paste here :)

Let's set the scope set to:
 * `queue:cancel-task`
 * `assume:scheduler-id:<schedulerId>/<taskGroupId>`
Naturally this also changes the parameters to the `req.satisfies` call.

The scope `assume:scheduler-id:<schedulerId>/<taskGroupId>` is also required to rerun a task. So I think this is an appropriate granularity. After all this end-point is supposed to be used by _schedulers_ that want's to cancel a task they have scheduled.